picard/file.py

```diff
@@ -250,6 +260,48 @@ def _loading_finished(self, callback, result=None, error=None):
         self.update()
         callback(self)
 
+    _types_split_re = re.compile('[^a-z0-9]', re.IGNORECASE)
+    _known_types = set([t['name'] for t in CAA_TYPES])
+    _default_types = ['front']
+
+    def get_types(self, string):
+        found = set([x.lower() for x in self._types_split_re.split(string) if x])
```

Comment:
Can also be done with a set comprehension `{x.lower() for x in self._types_split_re.split(string) if x}`